### PR TITLE
Rebase Pip to 21.2.1

### PR DESCRIPTION
**Pull request by cobaltgit**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,22 +15,24 @@ ARG BASE_IMAGE_NAME=debian:buster-slim
 # Intermediate build container
 FROM $BASE_IMAGE_NAME AS builder
 
-ENV PYTHON_VERSION 3.9.6
-ENV PYTHON_PIP_VERSION 21.1.3
+# Set Python and Pip versions
+ENV PYTHON_VERSION=3.9.6 \
```
Comment (eht16) [inline Dockerfile]: This is not necessary (as well as the other changes in 9431c4a0efd10eced93b2ffaf11d1c0accfabb6d). As already explained in https://github.com/eht16/python3-docker/pull/1#discussion_r666512786: this is a temporary build container and all its layers will be dropped and will not be included in the final image.

File: Dockerfile [hunk 2]
```diff
@@ -112,19 +117,19 @@ ENV LANG=C.UTF-8 \
...
 	ln -s /python/bin/python3 /usr/local/bin/python && \
 	ln -s /python/bin/python3 /usr/local/bin/python3 && \
 	ln -s /python/bin/pip3 /usr/local/bin/pip && \
 	ln -s /python/bin/pip3 /usr/local/bin/pip3 && \
-	# install depedencies
+	# Install dependencies
 	apt-get update && \
 	apt-get install --assume-yes --no-install-recommends ca-certificates libexpat1 libsqlite3-0 libssl1.1 && \
-	apt-get purge --assume-yes --auto-remove -o APT::AutoRemove::RecommendsImportant=false && \
+	apt-get purge --assume-yes --auto-remove -o APT::AutoRemove::RecommendsImportant=false && apt-get clean && \
```
Comment (eht16) [inline Dockerfile]: Does `apt-get clean` anything meaningful more than `rm -rf /var/lib/apt/lists/*`?

**Review by eht16 — CHANGES_REQUESTED** [review]
Thanks for the followup.
Would you mind checking my comments?

**Comment by eht16** [comment]
@CobaltGit Would you mind checking my comments?

**Comment by cobaltgit** [comment]
Sorry for the delay, I'll get to work now

**Comment by cobaltgit** [comment]
Also, are you alright with the image being pushed to a public registry (Docker Hub preferred)?

**Comment by eht16** [comment]
> Also, are you alright with the image being pushed to a public registry (Docker Hub preferred)?

Yes, sure. As long as I don't have to do it :).

**Comment by cobaltgit** [comment]
Oh don't worry, I'll take care of that

**Comment by cobaltgit** [comment]
I'll also throw in variants with other base images e.g Ubuntu Focal

**Comment by cobaltgit** [comment]
> I'll also throw in variants with other base images e.g Ubuntu Focal

https://hub.docker.com/repository/docker/cobaltdocker/eht16-python3/general

**Comment by eht16** [comment]
Nice!
You don't need to keep my name in the image name nor in the description. I don't mind at all :).

**Pull request merged**